evap/settings.py

```diff
@@ -352,6 +352,17 @@ class ManifestStaticFilesStorageWithJsReplacement(ManifestStaticFilesStorage):
 # Absolute filesystem path to the directory that will hold user-uploaded files.
 MEDIA_ROOT = os.path.join(BASE_DIR, "upload")
 
+### Evaluation progress rewards
+GLOBAL_EVALUATION_PROGRESS_REWARDS: list[tuple[float, str]] = [
+    (0, "0€"),
+    (0.25, "1.000€"),
+    (0.6, "3.000€"),
+    (0.7, "7.000€"),
+    (0.9, "10.000€"),
+]
+EXCLUDED_COURSE_TYPE_IDS: list[int] = []
+EXCLUDED_EVALUATION_IDS: list[int] = []
+PARTICIPANT_PROGRESS_TEXT = "Deine Teilnahme am Evaluationsprojekt wird helfen. Evaluiere also <b>jetzt</b>!"
```

Comment:
I'd want to mark this string as safe here, because here, the reader can reason about the contents. Once its passed through the view into the template, the line `{{ progress_description | safe }}` is hard for me to verify.

```suggestion
PARTICIPANT_PROGRESS_TEXT = mark_safe("Deine Teilnahme am Evaluationsprojekt wird helfen. Evaluiere also <b>jetzt</b>!")
```

(also: translation)